screens/LoadingNavigator/LoadingNavigator.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { View } from 'react-native'
+import { createStackNavigator } from '@react-navigation/stack'
+import { LinkingOptions, NavigationContainer } from '@react-navigation/native'
+import * as Linking from 'expo-linking'
+
+export interface LoadingParamList {
+  Loading: undefined
+
+  [key: string]: undefined | object
+}
+
+const LoadingStack = createStackNavigator<LoadingParamList>()
+
+export function LoadingNavigator (): JSX.Element {
+  return (
+    <NavigationContainer linking={LinkingConfiguration}>
+      <LoadingStack.Navigator screenOptions={{ headerShown: false }}>
+        <LoadingStack.Screen name='Loading' component={View} />
```

Comment:
Maybe you can reuse the existing LoadingComponent here?

Reply:
This is temporary to test behavior of react router, I don't actually use it yet. Will create another PR when we start to use it.